doc/api/packages.md

```diff
@@ -51,12 +51,13 @@ along with a reference for the [`package.json`][] fields defined by Node.js.
 ## Determining module system
 
 Node.js will treat the following as [ES modules][] when passed to `node` as the
-initial input, or when referenced by `import` statements within ES module code:
+initial input, or when referenced by `import` statements or `import()`
+expressions:
 
-* Files ending in `.mjs`.
+* Files with a `.mjs` extension.
```

Comment:
```suggestion
* Files with an `.mjs` extension.
```

Reply:
Hum I read that as `a dot M-J-S extension`, is there another name for `.`?

Reply:
The periods aren’t spoken in a sentence like this. In spoken English you’d say “an em jay ess extension.” This also tracks that I would think that most people consider the extension to be `mjs`, not `.mjs` (as in, the dot isn’t part of the extension) which might explain why the period isn’t spoken.